store/tx_manager.go

```diff
@@ -27,6 +27,7 @@ type TxManager interface {
 	CreateTx(to common.Address, data []byte) (*models.Tx, error)
 	ActivateAccount(account accounts.Account) error
 	MeetsMinConfirmations(hash common.Hash) (bool, error)
+	ContractLINKBalance(wr models.WithdrawalRequest) (assets.Link, error)
 	WithdrawLink(wr models.WithdrawalRequest) (common.Hash, error)
 	GetLinkBalance(address common.Address) (*assets.Link, error)
```

Comment:
We have inconsistent naming here:

1. ContractLINKBalance
2. WithdrawLink
3. GetLinkBalance.

Let's keep it all the same. Low hanging fruit is to change to `ContractLinkBalance` but I am indifferent. Completely understand the uppercase scenario.

Reply:
Thanks. I found a handy tool for this.

```
~/chainlink$ gorename -force -from '"github.com/smartcontractkit/chainlink/store".TxManager.WithdrawLink' -to WithdrawLINK                                                                                               
Renamed 7 occurrences in 4 files in 4 packages.                                                                                                                                                                                                
~/chainlink$ gorename -force -from '"github.com/smartcontractkit/chainlink/store".TxManager.GetLinkBalance' -to GetLINKBalance                                                                                           
Renamed 8 occurrences in 4 files in 4 packages.                                                                                                                                                                                                
```